src/components/Footer/index.js

```diff
@@ -0,0 +1,3 @@
+import Footer from "./Footer";
+
+export default Footer;
```

Comment:
why is imported also from here the footer? this approach is good but it's benefits are when you define an index by module.